pkg/controllers/metrics/pod/controller.go

```diff
@@ -100,19 +101,40 @@ var (
 	PodUnstartedTimeSeconds = opmetrics.NewPrometheusGauge(
 		crmetrics.Registry,
 		prometheus.GaugeOpts{
-			Namespace: "karpenter",
+			Namespace: metrics.Namespace,
 			Subsystem: metrics.PodSubsystem,
 			Name:      "unstarted_time_seconds",
 			Help:      "The time from pod creation until the pod is running.",
 		},
 		[]string{podName, podNamespace},
 	)
+	PodUnscheduledDurationSeconds = opmetrics.NewPrometheusGauge(
+		crmetrics.Registry,
+		prometheus.GaugeOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: metrics.PodSubsystem,
+			Name:      "unscheduled_time_seconds",
+			Help:      "The time that a pod hasn't been bound since Karpenter first considered it schedulable",
+		},
+		[]string{podName, podNamespace},
+	)
+	PodScheduledDurationSeconds = opmetrics.NewPrometheusHistogram(
+		crmetrics.Registry,
+		prometheus.HistogramOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: metrics.PodSubsystem,
+			Name:      "scheduled_duration_seconds",
+			Help:      "The time it takes for pods to be bound from when it's first considered for scheduling in memory",
+		},
+		[]string{},
```

Comment:
Any thought around adding the pod name and namespace in these metrics?

Reply:
This is a histogram, so you want it to just be a function of all the pods, right?